ignite/templates/app/files/{{protoDir}}/buf.gen.gogo.yaml

```diff
@@ -3,16 +3,15 @@
 #
 # buf.gen.gogo.yaml
 #
-version: v1
+version: v2
 plugins:
-  - name: gocosmos
+  - local: protoc-gen-gocosmos
     out: .
     opt:
       - plugins=grpc
-      - Mgoogle/protobuf/any.proto=github.com/cosmos/gogoproto/types/any
+      - Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types
```

Comment:
We should still import `github.com/cosmos/gogoproto/types/any` and not `codec/types`